tests/test_helper.js

```diff
@@ -108,11 +108,18 @@ function assertValidHTML(response, done) {
                 return false;
             })
             .filter(function(e) {
-                console.log('>> e:', e);
-                if (e.match(/^Error: Attribute.+color.+not allowed on element.+link.+at this point./)) {
+                var ignoreRe = /^Error: Attribute.+color.+not allowed on element.+link.+at this point./;
+                var ignoreRe2 = /^Error: A.+link.+element with a.+sizes.+attribute must have a.+rel.+attribute that contains the value.+icon.+./;
```

Comment:
Let me have a look again later. I see this is merged https://github.com/w3c/html/pull/757 so the validator needs to be updated.